lib/riax.ex

```diff
@@ -1,4 +1,5 @@
 defmodule Riax do
```

Comment:
some top level module doc would be good. It's not clear to me what exactly is the job of this module. the fact that it has key/value sounds like you are assuming a key value store as the application